fs_src/script.js

```diff
@@ -1,19 +1,3 @@
-var lastInfo = null;
-
-var socket = null;
-
-var wifiEn = el("wifi_en");
```

Comment:
We use global element variables only for wifi and for hap, why? I think we should do it no where, use the same coding style every where.

Reply:
legacy. you are right, should just use `el()`